pyproject.toml

```diff
@@ -0,0 +1,217 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools", "wheel"]
+
+[project]
+name = "rdmo"
+description = "RDMO is a tool to support the systematic planning, organisation and implementation of the data management throughout the course of a research project."
+readme = "README.md"
+keywords = [
+  "data management plan",
+  "dmp",
+  "rdmo",
+  "research data",
+  "research data management",
+]
+license = {text = "Apache-2.0"}
+authors = [
+  {name = "RDMO Arbeitsgemeinschaft", email = "rdmo-team@listserv.dfn.de"},
+]
+requires-python = ">=3.8"
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Web Environment",
+  "Framework :: Django :: 4.2",
+  "Intended Audience :: Science/Research",
+  "License :: OSI Approved :: Apache Software License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+]
+dynamic = [
+  "version",
+]
+dependencies = [
+  "defusedcsv~=2.0",
+  "defusedxml~=0.7",
+  "django~=4.2",
+  "django-cleanup~=8.0",
+  "django-compressor~=4.4",
+  "django-extensions~=3.2",
+  "django-filter~=23.2",
+  "django-libsass~=0.9",
+  "django-mathfilters~=1.0",
+  "django-mptt~=0.14",
+  "django-rest-swagger~=2.2",
+  "django-settings-export~=1.2",
+  "django-split-settings~=1.2",
+  "django-widget-tweaks~=1.5",
+  "djangorestframework~=3.14",
+  "drf-extensions~=0.7",
+  "iso8601~=2.0",
+  "markdown~=3.4",
+  "pypandoc~=1.11",
+  "rules~=3.3",
+]
+
+[project.optional-dependencies]
+allauth = [
+  "django-allauth~=0.56",
+]
+ci = [
+  "coveralls~=3.3",
+  "rdmo[dev]",
+]
+dev = [
+  "django-debug-toolbar~=4.2",
+  "pre-commit~=3.4",
+  "rdmo[allauth]",
+  "rdmo[pytest]",
+]
+gunicorn = [
+  "gunicorn~=21.2",
+]
+ldap = [
+  "django-auth-ldap~=4.5",
+]
+memcached = [
+  "python-memcached~=1.5",
```

Comment:
Think that we should update this memcached depency as well:
For Django 4.2 and 3.2:
> There are several Python Memcached bindings available; the two supported by Django are [pylibmc](https://pypi.org/project/pylibmc/) and [pymemcache](https://pypi.org/project/pymemcache/).

According to https://docs.djangoproject.com/en/3.2/topics/cache/:
> Changed in Django 3.2:
> The PyMemcacheCache backend was added.
> 
> Deprecated since version 3.2: The MemcachedCache backend is deprecated as python-memcached has some problems and seems to be unmaintained. Use PyMemcacheCache or PyLibMCCache instead.
>

Reply:
Oh, we don't use caching anymore, this dependency can go.

Reply:
alright, was already wondering why it didn't break before with 3.2.
I will just remove it here then with a commit.

Reply:
this `LocMemCache` from `settings.py` is also not used or?